docs/troubleshooting-guide.md

```diff
@@ -50,3 +50,47 @@ from `provided` to `compile`.
 SystemML can then be rebuilt with the `commons-math3` dependency using
 Maven (`mvn clean package -P distribution`).
 
+## OutOfMemoryError in Hadoop Reduce Phase 
```

Comment:
I believe this file has been pulled in from a previous commit due to merging as well.